### ISSUE TITLE
Improvements

### ISSUE THREAD

**Issue by XapedemakX** [FoxP/CATALOGUE#1]
Hey FoxP  :) 

i want to know if a merge Tool is planned ? i need to have two differents catalogues but i want to merge them periodically.

second improvement : Possibility to add some sheets in Favorite ?

**Comment by FoxP** [comment]
Hi, a merge tool is planned for next release, i just need to do some final tests.
To add sheets to favorites, did you try the "CTRL + D" shortcut ? 
Or maybe you want to "pin" some sheets at the top of the HTML catalog ?

**Comment by XapedemakX** [comment]
Yes it's this, i want to pin some sheets at the top of the catalog :)
